src/soap/client.py

```diff
@@ -19,14 +21,22 @@ def build_client(
     and configured on the session, which is then used as transport for the zeep client.
 
     Any additional kwargs are passed through to the :class:`zeep.Client` instantiation.
-
-    .. todo:: Incorporate `WS-Security <https://docs.python-zeep.org/en/master/wsse.html>`_
-       concepts (see Chris' PR for Suwinet).
     """
     session_factory = SessionFactory(service)
     session = SOAPSession.configure_from(session_factory)
-    transport = transport_factory(session=session)
-    client = client_factory(service.url, transport=transport, **kwargs)
+    transport = transport_factory(
+        session=session,
+        timeout=settings.DEFAULT_TIMEOUT_REQUESTS,
+        # operation_timeout gets passed as a parameter on all requests, overriding any
+        # monkeypatched requests.Session defaults
+        operation_timeout=settings.DEFAULT_TIMEOUT_REQUESTS,
```

Comment:
hrm, maybe it's a good idea to update out monkeypatch to not only do a `setdefault` but actually check if the value `is None` and then apply it still?

If we go that route, we also need to update default project.